### PR TITLE
Remove reference info for Checkout schema

### PR DESCRIPTION
**Pull request by keharper**
## Description

This PR removes the reference information and adds links to the SpectaQL docs for the Checkout schema. I also added filler text for the index pages.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "@adobe/gatsby-theme-aio": "4.8.3",
-    "gatsby": "4.25.5",
+    "gatsby": "4.22.0",
```
Comment (jeff-matthews) [inline package.json:11]: Was there a reason for downgrading gatsby?

File: src/pages/graphql/schema/checkout/mutations/handle-payflow-pro-response.md [hunk 2]
```diff
@@ -63,24 +67,3 @@ mutation {
...
-
-## Output attributes
-
-The PayflowProResponseOutput contains a `Cart` object.
-
-### Cart object
-
-import CartObject from '/src/pages/_includes/graphql/cart-object-24.md'
-
-<CartObject />
```
Comment (jeff-matthews) [inline src/pages/graphql/schema/checkout/mutations/handle-payflow-pro-response.md:84]: I reckon other files are using this include. Is the plan to do a sweep after you've finished linking to the reference for the entire API?

If so, will you please add a story to your epic about removing the exclude pattern from the `test:links` npm script when that's done?

File: src/pages/graphql/schema/checkout/queries/customer-payment-tokens.md [hunk 3]
```diff
@@ -60,12 +64,6 @@ query {
 }
 ```
 
-## Output attributes
-
-import CustomerPaymentTokens from '/src/pages/_includes/graphql/customer-payment-tokens.md'
```
Comment (jeff-matthews) [inline src/pages/graphql/schema/checkout/queries/customer-payment-tokens.md:65]: Same question about include as previous comment

**Review by jeff-matthews — COMMENTED** [review]
Just a few questions

File: package.json (hunk 1 above)
Comment (bdenham) [reply package.json:11]: @jeff-matthews: Yes. The `aio-theme` does not support gatsby versions beyond 4.22.0. This was an issue I brought up last year, but was promptly closed for some reason. See https://jira.corp.adobe.com/browse/DEVSITE-532.

Comment (jeff-matthews) [reply package.json:11]: Thanks @bdenham. I totally forgot about this.

File: src/pages/graphql/schema/checkout/mutations/handle-payflow-pro-response.md (hunk 2 above)
Comment (keharper) [reply src/pages/graphql/schema/checkout/mutations/handle-payflow-pro-response.md:84]: It's [COMDOX-558](https://jira.corp.adobe.com/browse/COMDOX-558)

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
Looks like you need to add `#gitleaks:allow` next to the token examples to address the following GITLEAKS linting errors:

- https://github.com/AdobeDocs/commerce-webapi/actions/runs/4379299329/jobs/7665097571#step:6:121
- https://github.com/AdobeDocs/commerce-webapi/actions/runs/4379299329/jobs/7665097571#step:6:146

**Review by jeff-matthews — APPROVED** [review]

**Pull request merged**